ui/app/src/services/configuration.ts

```diff
@@ -95,6 +108,43 @@ export function getPreviewToolsConfig(site: string): Observable<PreviewToolsConf
   );
 }
 
+export function getAudiencesPanelConfig(site: string): Observable<AudiencesPanelConfig> {
```

Comment:
Notice the "region" comments on 35 and 189. Create a region for your module within this file outside of the Preview config region.